### PR TITLE
Addition of Afrikaans (af) language translation (with spec tests)

### PR DESCRIPTION
**Pull request by sarelvdwalt**
Added the Afrikaans (af) language (a derivative of Dutch - used in South Africa) yml file and wrote the tests to confirm that all is well with it ;)

### CODE REVIEW

**Comment by sarelvdwalt** [comment]
I also added a CONTRIBUTING.md file to help future contributors get into it quicker

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#A quick guide to contribute to the project:
+
+##Installing the dev environment
+
+1.  For the repo
```
Comment (norzechowicz) [inline CONTRIBUTING.md]: missing "r" in Fork :P

**Comment by norzechowicz** [comment]
@sarelvdwalt perfect! This is how PR should looks! :beers:

**Comment by sarelvdwalt** [comment]
Thanks @norzechowicz :) I fixed the typo there, sorry about that :)

**Pull request merged**